packages/utils/odsp-doclib-utils/package.json

```diff
@@ -45,7 +45,7 @@
 		}
 	},
 	"main": "dist/index.js",
-	"types": "dist/odsp-doclib-utils-public.d.ts",
+	"types": "dist/index.d.ts",
```

Comment:
@tylerbutler - Just FYI - I used the web editor to also trim the 'types' field on line 48.

Reply:
Thanks! I completely missed that.